components/composer/src/Composer.svelte

```diff
@@ -975,6 +975,22 @@
             <span class="sr-only">Save Draft</span>
           </button>
         {/if}
+        {#if _this.show_attachment_button && (id || uploadFile)}
+          <label
+            for="file-upload"
+            class="composer-btn file-upload"
+            title="Attach Files (up to 4MB)"
+            tabindex="0"
+          >
+            <AttachmentIcon class="FooterIcon" />
+            <span class="sr-only">Attach Files</span>
+          </label>
+          {#if _this.show_max_file_size}
+            <div class="file-size">
+              <span>Max file size: 4MB</span>
```

Comment:
Let's not hardcode this...can we also make the max file size a prop?

Reply:
@AaronDDM Sure we can, but isn't the api has a hardcoded maximum 4MB size?

Reply:
Okay, so @yifanplanet we do allow customers to set their own `uploadFile` function:
https://github.com/nylas/components/blob/b98c13f6a872777cf22b9785e32d112e6788db3b/components/composer/src/Composer.svelte#L288

This means that the upload file size limitation might not be true for our customer's implementation.

However, it looks like in our code we make that assumption:
https://github.com/nylas/components/blob/b98c13f6a872777cf22b9785e32d112e6788db3b/components/composer/src/Composer.svelte#L283

So for this story:
1. Can you make the max file size configurable?
2. Update #L283 as well with this configurable value.

Reply:
@AaronDDM All right it makes sense. And we should disable this file size check only if `uploadFile` callback function is used by user. For `nylasUploadFile` we should always have a maximum file size less than 4MB.

Reply:
That's true, technically a customer can add file size checks on their own if they use the `beforeUpload` function, I think. 

Or we allow someone to disable a file size check (e.g. supply a minimum file size of -1).

Let me know which approach you think works the best.